docs/en/siem/siem-ui.asciidoc

```diff
@@ -86,15 +87,14 @@ Interactive widgets let you drill down for deeper insights:
 
 There are also tabs for viewing and investigating specific types of data:
 
-* Flows
-** Source IPs
-** Destination IPs
-** Source Countries
-** Destination Countries
-* DNS
-* TLS
-* Anomalies
-* <<det-engine-terminology, External alerts>>
+* Flows: source and destination IP addresses and countries
+* DNS: DNS network queries
+* HTTP: received HTTP requests (HTTP requests for applications using
+{kibana-ref}/apm-getting-started.html[Elastic APM] are monitored by default)
```

Comment:
No worries since this is already merged, but we have the `{apm-app-ref}` so that when the APM content moves out of Kibana, we only have to update the link definition, instead of all the references that are scattered throughout the documentation.

Reply:
ok - i'll change that today. it might save us some hassle in the future...

Reply:
Thanks. Sorry about all the trouble 😅